go/protomodule/protomodule_test.go

```diff
@@ -725,6 +725,17 @@ func mustProtoMessage(t *testing.T, v starlark.Value) proto.Message {
 	return nil
 }
 
+func mustMarshalAny(t *testing.T, v proto.Message) []byte {
```

Comment:
This is kinda weird. Why return just the "value" and not the Any itself directly?